### DISCUSSION TITLE
Mend Dashboard: Display renovate cli version

### DISCUSSION THREAD

**Discussion by setchy** [renovatebot/renovate#32753] [Suggest an Idea]
### Tell us more.

It would be helpful if the Mend Dashboard (https://developer.mend.io/) displayed the version that the hosted GitHub Renovate App.

This would save digging into a recent repository log file

**Comment by setchy** [comment]
In addition, it would be helpful if the renovate version was displayed in the header of each job run

<img width="953" alt="Screenshot 2024-11-28 at 9 42 31 AM" src="https://github.com/user-attachments/assets/d9db9411-d156-4885-a9a1-2112480d6906">

**Comment by justo-mend** [comment]
Thanks for your suggestions. We've taken your idea and are looking into it.
My first thought was to show the current Renovate version on the Org page - top right corner, directly below the Renovate Engine Status, in a slightly smaller font.

![image](https://github.com/user-attachments/assets/bb44290a-f6b2-4c4f-9a0a-3a7813d8ee6a)

**Comment by justo-mend** [comment]
Thanks for adding the suggestion to also provide the Renovate version on the job log page. This would not necessarily be the "current" version, but instead the version that was used when the job was run. Nice idea.

**Reply by setchy** [reply]
That's right. Renovate version at time of job

**Reply by justo-mend** [reply]
I'll create a separate ticket for this.

**Comment by justo-mend** [comment]
One issue: For users running the full Mend Platform, they would probably also want to see the version of the SAST and SCA scanners. So we might need to find a place to show them all.

**Reply by justo-mend** [reply]
Maybe we could introduce an info page, similar to the "Help->About" section of most desktop apps, that shows all the relevant info for the Developer Portal.
It could include version for:
- Controller
- Front end
- SCA Scanner
- SAST Scanner
- Renovate
And anything else that's pertinent.

Question for you: Is it important to see the versions without having to open a new page/tab/pop-up? (ie. always shown on the Org page or application header)

**Reply by setchy** [reply]
Perhaps both... Show the main version inline with more detailed info in a subpage

**Comment by setchy** [comment]
This is now available in the mend dashboard
